tests/cases/fourslash/renameNumericalIndex.ts

```diff
@@ -0,0 +1,6 @@
+/// <reference path="fourslash.ts" />
+
+////const foo = { 0: true };
+////foo[[|{| "contextRangeIndex": 0 |}0|]];

```

Comment:
What does `contextRangeIndex: 0` mean?

Reply:
Not sure, I was following the code from other rename test cases like the ones in the `renameModuleExportsProperties` files.

Reply:
Can you see if the test still seems to work without it?